docs/sources/menu.yaml

```diff
@@ -260,26 +260,26 @@
     - name: Variable types
       link: /variables/variable-types/
       children:
-      - link: /variables/variable-types/add-query-variable/
-        name: Add query variable
-      - link: /variables/variable-types/add-custom-variable/
-        name: Add custom variable
-      - link: /variables/variable-types/add-text-box-variable/
-        name: Add text box variable
-      - link: /variables/variable-types/add-constant-variable/
-        name: Add constant variable
-      - link: /variables/variable-types/add-data-source-variable/
-        name: Add data source variable
-      - link: /variables/variable-types/add-interval-variable/
-        name: Add interval variable
-      - link: /variables/variable-types/add-ad-hoc-filters/
-        name: Add ad hoc filters
-      - link: /variables/variable-types/chained-variables/
-        name: Chained variables
-      - link: /variables/variable-types/global-variables/
-        name: Global variables
-      - link: /variables/variable-types/url-variables/
-        name: URL variables
+        - link: /variables/variable-types/add-query-variable/
```

Comment:
@marcusolsson, are these indents on purpose? The menu was building correctly before.

Reply:
Yes. While there's no semantic difference, the indentations were inconsistent with the rest of the document.

Compare

```
- name: Developers
  children:
    - name: Plugins
``` 

with

```
- name: Developers
  children:
  - name: Plugins
```